heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -219,6 +226,13 @@ private static Options constructOptions() {
         .required()
         .build();
 
+    Option dryRun = Option.builder("u")
```

Comment:
We really need to refactor so we can stop copying code between this class and RuntimeManagerMain. Can we do that first, in a separate PR?